### DISCUSSION TITLE
[Hubs Cloud]media frame can not snap media (pdf, webcam ,screen)

### DISCUSSION THREAD

**Discussion by suchart28** [mozilla/hubs#5354] [Hubs Cloud Questions]
Spoke app
1. add media frame with scence
2. Publish scence

Chrome

1. create room with scence https://suchartstudio.com/scenes/z8FZMrB
2. try media pin to frame can not snap media to frame

Device
Windows 11
Chrome

link : https://suchartstudio.com/scenes/z8FZMrB

**Comment by matthewbcool** [comment]
I was unable to reproduce on my hubs cloud, can you share a room link we can test in?
